components/dashboards/org.wso2.carbon.dashboards.core/src/main/resources/sql/mysql.sql

```diff
@@ -12,11 +12,11 @@ CREATE TABLE IF NOT EXISTS DASHBOARD_RESOURCE (
   UNIQUE (ID)
 );
 
-DROP TABLE WIDGET_RESOURCE;
+DROP TABLE IF EXISTS WIDGET_RESOURCE;
 CREATE TABLE IF NOT EXISTS WIDGET_RESOURCE (
   WIDGET_ID                   VARCHAR(255) NOT NULL,
   WIDGET_NAME                 VARCHAR(255) NOT NULL,
   WIDGET_CONFIGS              LONGBLOB,
   CONSTRAINT PK_WIDGET_RESOURCE PRIMARY KEY (WIDGET_ID, WIDGET_NAME),
-  UNIQUE (WIDGET_NAME, WIDGET_NAME)
+  UNIQUE (WIDGET_ID, WIDGET_NAME)
```

Comment:
This needs to be fixed in [h2](https://github.com/wso2/carbon-dashboards/blob/v4.0.15/components/dashboards/org.wso2.carbon.dashboards.core/src/main/resources/sql/h2.sql#L20) as well.

Reply:
Fix added with commit fd38301d7c91c5d30f403edad955e8fd1a22ce71